python/rmm/CMakeLists.txt

```diff
@@ -30,4 +30,5 @@ rapids_cython_init()
 add_compile_definitions("SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_LOGGING_LEVEL}")
 
 add_subdirectory(rmm/_cuda)
```

Comment:
I'm open to different names than `python` and `cpp`.  And changing `cpp` to `_cpp`

Reply:
In pylibcudf all the pxds containing C++ bindings are in [a libcudf subpackage](https://github.com/rapidsai/cudf/tree/branch-24.10/python/pylibcudf/pylibcudf/libcudf). How about we match that with cpp->librmm? It's not immediately obvious that there will or should be a pylibrmm/rmm split, but if there is that could be done in a follow-up to keep this PR's scope limited.